x-pack/plugins/enterprise_search/common/types/workplace_search.ts

```diff
@@ -30,49 +30,3 @@ export interface ConfiguredLimits {
     totalFields: number;
   };
 }
-
-export interface Group {
```

Comment:
@scottybollinger FYI on 08057fa86759dcf74d0ce90996015661de257a18 - I moved these types out of `common/` and into `public/` since there isn't any actual logic in `server/` that needs those types. If you're not a fan of the change no worries, LMK and I can drop/revert this commit